src/sentry/conf/server.py

```diff
@@ -104,7 +104,7 @@ def env(key, default='', type=None):
         'NAME': 'sentry',
         'USER': 'postgres',
         'PASSWORD': '',
-        'HOST': '',
+        'HOST': '127.0.0.1',
```

Comment:
Changing the default value here to not use a unix socket, and this should be backwards compatible in every way.